assets/css/style.css

```diff
@@ -117,6 +117,18 @@ h1, h2, h3, h4, h5, h6 {
     display: none;
 }
 
+section .alert-success {
+    margin-top: 15px;
+    position: absolute;
+    background-color: #ccf6e4;
+    border-color: #b8f2da;
+    color: #006d3f;
+    width: 100%;
+    text-align: center;
+    z-index: 99;
+    display: none;
+}
+
```

Comment:
This here is the same code as a couple of lines above. 

A better solution would be to remove `section` and `#header` from `.alert-success` and keep just one CSS class.

This will apply the style to all HTML elements containing `.alert-success` CSS class.

![mroomapp – style css 2021-11-20 16-11-38](https://user-images.githubusercontent.com/12160621/142729532-154a47ea-f1f2-4228-a9a1-111ec04ab166.png)

Once you do that, you have to make sure nothing else breaks. 
(It might be `#header .alert-success`is used as CSS selector somewhere)

Search for code usage and modify to use only `.alert-success`:

![Monosnap 2021-11-20 16-13-51](https://user-images.githubusercontent.com/12160621/142729520-db2d7633-c1ec-4fdf-a613-7a6d785700b0.png)